services/api/src/main/java/io/swec/services/api/Main.java

```diff
@@ -21,8 +21,8 @@
     public static final String BASE_URI = "http://0.0.0.0:9081/myapp/";
 
     // Sonar issue
-    // String uname = "steve";
-    // String password = "blue";
+     String uname = "steve";
+     String password = "blue";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'password' detected in this expression, review this potentially hard-coded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://165.22.210.192:9000/coding_rules#rule_key=squid%3AS2068)